pkgs/by-name/cs/csharpier/package.nix

```
@@ -0,0 +1,19 @@
+{ buildDotnetGlobalTool, lib }:
+
+buildDotnetGlobalTool {
+  pname = "csharpier";
+  version = "0.26.4";
+  executables = "dotnet-csharpier";
+
+  nugetSha256 = "sha256-9ieSAWulfJZz0INRf95Nz5C865sM8kffK2UGZ1NZ6Z0=";
+
+  meta = with lib; {
+    description = "CSharpier is an opinionated code formatter for C#";
```

I think the description needs a small touch: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes

```suggestion
    description = "An opinionated code formatter for C#";
```